src/frontend/apps/e2e/__tests__/app-desk/member-delete.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { expect, test } from '@playwright/test';
+
+import { addNewMember, createTeam, keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Members Delete', () => {
+  test('it deletes himself when it is the last owner', async ({
+    page,
+    browserName,
+  }) => {
+    await createTeam(page, 'member-delete-1', browserName, 1);
+
+    const table = page.getByLabel('List members card').getByRole('table');
+
+    const cells = table.getByRole('row').nth(1).getByRole('cell');
+    await expect(cells.nth(1)).toHaveText(
+      new RegExp(`E2E ${browserName}`, 'i'),
+    );
+    await cells.nth(4).getByLabel('Member options').click();
+    await page.getByLabel('Open the modal to delete this member').click();
+
+    await expect(
+      page.getByText(
+        'You are the last owner, you cannot be removed from your team.',
+      ),
+    ).toBeVisible();
+    await expect(page.getByRole('button', { name: 'Validate' })).toBeDisabled();
+  });
+
+  test('it deletes himself when it is not the last owner', async ({
+    page,
+    browserName,
+  }) => {
+    await createTeam(page, 'member-delete-1', browserName, 1);
+
+    await addNewMember(page, 0, 'Owner');
+
+    const table = page.getByLabel('List members card').getByRole('table');
+
+    // find row where regexp match the name
+    const cells = table
+      .getByRole('row')
+      .filter({ hasText: new RegExp(`E2E ${browserName}`, 'i') })
+      .getByRole('cell');
+    await cells.nth(4).getByLabel('Member options').click();
+    await page.getByLabel('Open the modal to delete this member').click();
+
+    await page.getByRole('button', { name: 'Validate' }).click();
+    await expect(
+      page.getByText(`The member has been removed from the team`),
+    ).toBeVisible();
+    await expect(
+      page.getByRole('button', { name: `Create a new team` }),
+    ).toBeVisible();
+  });
+
+  test('it deletes owner member', async ({ page, browserName }) => {
```

Comment:
`it cannot delete another owner member`

Reply:
https://github.com/numerique-gouv/people/pull/147/commits/f4583b546c820e8cd6c5ca74732c6b7c46dee59a